src/jvmTest/kotlin/wizard/GeneratedComposeAppProjectTest.kt

```diff
@@ -164,8 +192,8 @@ class GeneratedComposeAppProjectTest {
                 "Debug",
                 "OBJROOT=${dir.path}/build/ios",
                 "SYMROOT=${dir.path}/build/ios",
-                "-sdk",
-                "iphonesimulator",
```

Comment:
"-sdk iphonesimulator" doesn't work for some reasons if I launch tests on my local machine. But "-destination id=..." works well locally and on github actions.